soc/espressif/esp32s3/memory.h

```diff
@@ -78,8 +78,8 @@
 
 /* AMP */
 #if defined(CONFIG_SOC_ENABLE_APPCPU) || defined(CONFIG_SOC_ESP32S3_APPCPU)
-#define APPCPU_IRAM_SIZE CONFIG_ESP32S3_APPCPU_IRAM_SIZE
-#define APPCPU_DRAM_SIZE CONFIG_ESP32S3_APPCPU_DRAM_SIZE
+#define APPCPU_IRAM_SIZE CONFIG_ESP32_APPCPU_IRAM_SIZE
```

Comment:
is this really right? (changed from ESP32S3 to ESP32)

Reply:
yes, since the AMP kconfig is being moved to the `common` folder it is no longer SoC specific, therefore rename.

Reply:
right. Naming is sometimes tricky, but I'd think about one that doesn't seem like a direct mention to a device.
Maybe ESP instead of ESP32

Reply:
Yes, that is moved to common folder which applies for all AMP capable targets. I'll take a look if we can use `ESP` instead of `ESP32`.